packages/lit-renderer/src/lit-renderer.d.ts

```diff
@@ -0,0 +1,14 @@
+import { RenderOptions } from 'lit';
+import { Directive } from 'lit/directive.js';
+
+export abstract class LitRendererDirective<E extends Element, R extends LitRenderer> extends Directive {
```

Comment:
There should be an export of the actual `LitRenderer` type, did you miss to add it?

Reply:
Good catch, thanks, I'll add it soon. By the way, the PR is not completely ready yet, particularly type definitions.

Reply:
There are some missing tests, in particular.

Reply:
Done.